anaSbitReadout.py

```diff
@@ -0,0 +1,376 @@
+#!/bin/env python
+"""
+    Script to analyze sbitreadout data
+    By: Caterina Aruta (caterina.aruta@cern.ch) and Francesco Ivone (francesco.ivone@cern.ch)
+    """
+if __name__ == '__main__':
+    import ROOT as r
```

Comment:
Please address [this comment](
https://github.com/cms-gem-daq-project/vfatqc-python-scripts/pull/208#discussion_r226918729) from @jsturdy